src/api/extrinsic/index.ts

```diff
@@ -65,26 +65,27 @@ export const createExtrinsicHex = (
 };
 
 /**
- * Recreates an extrinsic from its hex-encoded representation.
+ * Reconstructs a SubmittableExtrinsic from a hex-encoded extrinsic string.
+ * The method is extracted and wrapped in a new submittable extrinsic.
  *
- * @param {ApiPromise} api - The Polkadot API instance.
- * @param {string} extrinsicHex - Hex-encoded string of the SubmittableExtrinsic.
- * @returns {SubmittableExtrinsic<'promise'>} The reconstructed SubmittableExtrinsic.
- * @throws {Error} - Throws an error if the reconstruction from hex fails.
+ * Note: This does not preserve the original signature.
+ *
+ * @param api - The Polkadot API instance.
```

Comment:
Is there a reason why the type annotation was removed from the `api` field? Same for the other const parameters.